src/Datadog.Trace.ClrProfiler.Managed/InterceptMethodAttribute.cs

```diff
@@ -59,6 +59,15 @@ public class InterceptMethodAttribute : Attribute
         /// </summary>
         public string TargetSignature { get; set; }
 
+        /// <summary>
+        /// Gets or sets the explicit type array for the target method to be intercepted.
+        /// This is a required field.
+        /// Must match the wrapper method in count:
```

Comment:
Is there additional documentation describing the order of the elements in this type array? i.e. return type followed by target method parameter types